v3/newrelic/expect_implementation.go

```diff
@@ -228,25 +228,25 @@ func expectLogEvents(v internal.Validator, events *logEvents, expect []internal.
 	}
 }
 
-func expectLogEvent(v internal.Validator, event logEvent, want internal.WantLog) {
-	if event.message != want.Message && want.Message != internal.MatchAnyString {
-		v.Error(fmt.Sprintf("unexpected log message: want %s, got %s", event.message, want.Message))
+func expectLogEvent(v internal.Validator, actual logEvent, want internal.WantLog) {
```

Comment:
Calling this `actual` instead of `event` is a good thing here to clarify the test cases.

Reply:
it confused me for the last time haha